deploy_to_maven_central.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#------------------------------------------------------------
+## @todo - test with gpg key with no passphrase so key provided by peter schrammel can be used
+
+## note - as a file with plain text is generated - it would be best if these were placed in a location
+## external to the repository, so they can't be checked in accidentally. Travis would also be good - all data is volatile.
```

Comment:
Add these files to gitignore to reduce chance of accidental checkin

Reply:
Done.